packages/interfaces/src/connection_manager.ts

```diff
@@ -63,7 +85,8 @@ export interface IConnectionStateEvents {
 
 export interface IConnectionManager
   extends TypedEventEmitter<IPeersByDiscoveryEvents & IConnectionStateEvents> {
-  configuredPubsubTopics: PubsubTopic[];
+  pubsubTopics: PubsubTopic[];
+  getConnectedPeers(codec?: string): Promise<Peer[]>;
```

Comment:
let's use the `Protocols` enum instead of the codec string here -- easier way to get information from the function without expecting the knowledge of exact codec (we already export the `Protocols` enum)